Osmalyzer/Analyzers/Misc Analyzers/ImproperTranslationAnalyzer.cs

```diff
@@ -213,6 +267,24 @@ public override void Run(IReadOnlyList<AnalysisData> datas, Report report)
                 )
             );
         }
+    
+            
+        report.AddGroup(
+            ReportGroup.OtherLanguages, 
+            "Other languages",
+            "This lists entries of languages that were not checked. "
+        );
+
+        foreach (var (locale, number) in ignoredLocales)
+        {
+            report.AddEntry(
+                ReportGroup.OtherLanguages,
+                new IssueReportEntry(
+                    "Locale '" + locale + "' was ignored. " + 
+                    number + " element" + (number % 10 == 1 ? "" : "s") + " have tag `name:" + locale + "`"
```

Comment:
This is a nice bit of formatting, but in English it's actually "21 items" ;)

Reply:
Whoops. Used this incorrectly for years now(